### DISCUSSION TITLE
Change checkbox shape?

### DISCUSSION THREAD

**Discussion by mitten** [Akifyss/obsidian-border#211] [Q&A] [answered]
I've looked through all the settings, and I'm sure I've just missed it somewhere, but I can't see where to change the shape of the checkboxes from round to square. Can someone point me at where to change the box shape? And if there isn't a setting, does anyone know which CSS selectors to use to change it? Thanks so much!

**Comment by Akifyss** [comment, marked as answer]
You can import the following CSS snippet to make it square:
```css
body {
    --checkbox-radius: 4px;
}
```
